wayang-platforms/wayang-spark/src/main/scala/README.md

```diff
@@ -0,0 +1,16 @@
+[comment]: # (Licensed to the Apache Software Foundation (ASF) under one or more)
+[comment]: # (contributor license agreements.  See the NOTICE file distributed with)
+[comment]: # (this work for additional information regarding copyright ownership.)
+[comment]: # (The ASF licenses this file to You under the Apache License, Version 2.0)
+[comment]: # ((the "License"); you may not use this file except in compliance with)
+[comment]: # (the License.  You may obtain a copy of the License at)
+[comment]: # ()
+[comment]: # (http://www.apache.org/licenses/LICENSE-2.0)
+[comment]: # ()
+[comment]: # (Unless required by applicable law or agreed to in writing, software)
+[comment]: # (distributed under the License is distributed on an "AS IS" BASIS,)
+[comment]: # (WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.)
+[comment]: # (See the License for the specific language governing permissions and)
+[comment]: # (limitations under the License.)
+
+In this folder it was created to avoid error in the parameters of the variables
```

Comment:
Please change to:
This directory was created to avoid errors with the parameters from variables